packages/amazonq/.vscode/launch.json

```diff
@@ -13,9 +13,9 @@
             "args": ["--extensionDevelopmentPath=${workspaceFolder}"],
             "env": {
                 "SSMDOCUMENT_LANGUAGESERVER_PORT": "6010",
-                "WEBPACK_DEVELOPER_SERVER": "http://localhost:8080"
-                // "__AMAZONQLSP_PATH": "${workspaceFolder}/../../../language-servers/app/aws-lsp-codewhisperer-runtimes/out/agent-standalone.js",
-                // "__AMAZONQLSP_UI": "${workspaceFolder}/../../../language-servers/chat-client/build/amazonq-ui.js"
+                "WEBPACK_DEVELOPER_SERVER": "http://localhost:8080",
+                "__AMAZONQLSP_PATH": "${workspaceFolder}/../../../language-servers/app/aws-lsp-codewhisperer-runtimes/out/agent-standalone.js",
```

Comment:
you can recomment these (or just uncommit the file) otherwise if devs don't have the language-server repos downloaded it's going to fail to launch

Reply:
Just gonna comment theses out again